tools/devtools/src/devtools_app/devtools.js

```diff
@@ -0,0 +1,79 @@
+import { isFirefox } from "../utils";
+
+let created = false;
+let checks = 0;
+let browserInstance = isFirefox() ? browser : chrome;
+let scriptsLoaded = false;
+
+// Trigger createPanelsIfOwl when the active tab changes to an other tab
+browserInstance.devtools.network.onNavigated.addListener(createPanelsIfOwl);
+
+// When the tab gets reloaded or url changes while devtools are oppened, try to reload the scripts
+// and send a reload message to the devtools owl app (relayed by the background thread)
+if (!isFirefox()) {
+  chrome.tabs.onUpdated.addListener((tab) => {
+    chrome.tabs.get(tab, (tabData) => {
+      if (tabData?.status === "complete") {
+        scriptsLoaded = false;
+        setTimeout(() => {
+          loadScripts().then(() => {
+            chrome.runtime.sendMessage({ type: "Reload" });
+          });
+        }, 100);
```

Comment:
The page having finished loading doesnt mean owl has also finished loading on the page so we need to wait. Needed to change to 200 ms because 100 ms was too short.